Network/DNS/IO.hs

```diff
@@ -45,7 +48,8 @@ receive :: Socket -> IO DNSMessage
 receive sock = do
     let bufsiz = fromIntegral maxUdpSize
     bs <- recv sock bufsiz `E.catch` \e -> E.throwIO $ NetworkFailure e
-    case decode bs of
+    Elapsed (Seconds now) <- timeCurrent
+    case decodeAt (fromIntegral now) bs of
```

Comment:
`fromIntegral` is not necessary.

Reply:
Yes, thanks. that'll be part of the review `fixup` commit.